test/gilded-rose.spec.ts

```diff
@@ -1,9 +1,190 @@
 import { Item, GildedRose } from '@/gilded-rose';
 
 describe('Gilded Rose', () => {
-  it('should foo', () => {
-    const gildedRose = new GildedRose([new Item('foo', 0, 0)]);
-    const items = gildedRose.updateQuality();
-    expect(items[0].name).toBe('fixme');
-  });
+  describe('Generic Items', () => {
+    it('should foo', () => {
+      const gildedRose = new GildedRose([new Item('foo', 0, 0)]);
+      const items = gildedRose.updateQuality();
+      expect(items[0].name).toBe('foo');
+    });
+    it('Given an item, when updateQuality is called, then sellin and quality lowered.', () => {
+      // Given
+      const testEnv = new GildedRose([new Item('item1', 5, 5)]);
```

Comment:
I'm not sure about the name `testEnv` - think `gildedRose` would be a perfectly fine name